tox.ini

```diff
@@ -13,8 +13,6 @@ deps =
     -rrequirements.txt
     -rdev-requirements.txt
 
-install_command = python -m pip install --no-deps {opts} {packages}
```

Comment:
We probably want to keep this so pip doesn't try to do a separate dependency resolution step.